Source/WebKit/UIProcess/WebAuthentication/Cocoa/LocalAuthenticator.mm

```diff
@@ -303,11 +303,11 @@ static inline uint8_t authDataFlags(ClientDataType type, LocalConnection::UserVe
 
     RetainPtr<SecAccessControlRef> accessControl;
     {
-        CFErrorRef errorRef = nullptr;
-        accessControl = adoptCF(SecAccessControlCreateWithFlags(NULL, kSecAttrAccessibleWhenPasscodeSetThisDeviceOnly, kSecAccessControlPrivateKeyUsage | kSecAccessControlUserPresence, &errorRef));
-        auto retainError = adoptCF(errorRef);
-        if (errorRef) {
-            receiveException({ ExceptionCode::UnknownError, makeString("Couldn't create access control: "_s, String(((NSError*)errorRef).localizedDescription)) });
+        CFErrorRef rawError = nullptr;
+        accessControl = adoptCF(SecAccessControlCreateWithFlags(NULL, kSecAttrAccessibleWhenPasscodeSetThisDeviceOnly, kSecAccessControlPrivateKeyUsage | kSecAccessControlUserPresence, &rawError));
```

Comment:
This function needs `CF_RETURNS_RETAINED` added to the last argument.
Otherwise, there is no way for static analyzer to know it's a +1 out argument.

Reply:
Yeah, that seems reasonable to me. Sorry, I forgot to actually approve it.